### PR TITLE
chore(bridge-ui-v2): update and fix types

### PR DESCRIPTION
**Pull request by monleru**
**- update and fix types**

### CODE REVIEW

**Comment by KorbinianK** [comment]
@monleru Thank you for your contribution, however you are changing two packages: starter dapp and bridge UI, which is not ideal.

Also this is no "fix" but a "chore. Please create separate PRs for each package in the future. But your changes to the starter dapp do not qualify as significant enough contribution, as we mentioned in the comment to your previous PR.

So please fix the title and revert your changes to the starter dapp :)

**Comment by monleru** [comment]
[KorbinianK](https://github.com/KorbinianK) Done

**Comment by KorbinianK** [comment]
Please fix the errors the pipeline shows: https://github.com/taikoxyz/taiko-mono/actions/runs/7301302271/job/19897670147?pr=15410

File: packages/bridge-ui-v2/src/libs/bridge/fetchTransactions.ts [hunk 1]
```diff
@@ -5,12 +5,12 @@ import { bridgeTxService } from '$libs/storage';
 import { getLogger } from '$libs/util/logger';
 import { mergeAndCaptureOutdatedTransactions } from '$libs/util/mergeTransactions';
 
-import { type BridgeTransaction, MessageStatus } from './types';
+import { type BridgeTransaction, MessageStatus,FetchTransactions } from './types';
```
Comment (d1onys1us) [inline packages/bridge-ui-v2/src/libs/bridge/fetchTransactions.ts:8]:
```suggestion
import { type BridgeTransaction, MessageStatus, FetchTransactions } from './types';
```

**Comment by monleru** [comment]
> @monleru Please fix the errors the pipeline shows: https://github.com/taikoxyz/taiko-mono/actions/runs/7301302271/job/19897670147?pr=15410

@KorbinianK Done

**Comment by KorbinianK** [comment]
> > @monleru Please fix the errors the pipeline shows: https://github.com/taikoxyz/taiko-mono/actions/runs/7301302271/job/19897670147?pr=15410
> 
> @KorbinianK Done

Linting still fails

**Comment by monleru** [comment]
> > > @monleru Please fix the errors the pipeline shows: https://github.com/taikoxyz/taiko-mono/actions/runs/7301302271/job/19897670147?pr=15410
> > 
> > 
> > @KorbinianK Done
> 
> Linting still fails

sorry, fixed @KorbinianK

**Comment by monleru** [comment]
@KorbinianK i added, check this pls

**Comment by KorbinianK** [comment]
> @KorbinianK i added, check this pls

Sorry for the delay, new years is a bit busy. Can you please rebase this to `alpha-6` to avoid conflicts. `alpha-6` is currently ahead.

**Comment by KorbinianK** [comment]
@monleru please do the rebase by end of the week, otherwise we will close this stale PR for now. You can always re-create a new one later.

**Pull request closed without merging**